src/repositories/likes.repository.ts

```diff
@@ -35,5 +35,5 @@ export const updateOne = (
   id: string,
   newData: { tweet?: string; user?: string }
 ) => {
-  LikeModel.updateOne({ _id: id }, newData);
+  return LikeModel.findOneAndUpdate({ _id: id }, newData);
```

Comment:
```suggestion
  return LikeModel.findOneAndUpdate({ _id: id }, newData, {
  new: true
});
```
You should set the new option to true to return the document after update was applied.
https://mongoosejs.com/docs/tutorials/findoneandupdate.html